kubeflow/trainer/rhai/transformers.py

```
@@ -281,15 +281,12 @@ def _async_checkpoint(self):
                 _log(f"Starting JIT checkpoint at step {current_step}")
 
-                # Get rank for distributed training. Fall back to True for single-process
-                # runs or if accelerate is unavailable.
+                # Get local rank 0 for distributed training.
+                # Fall back to True for single-process runs.
                 try:
-                    from accelerate import PartialState
-
-                    is_main_process = PartialState().is_main_process
+                    is_local_rank0 = self.trainer.args.local_process_index == 0
                 except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)